home/bin/gitlab

```diff
@@ -20,6 +20,7 @@ test -z "$DOCKER" && { echo "Error: Docker not installed." >&2; exit 2; }
 test -d "${ORE_HOME}" || { mkdir -p "${ORE_HOME}"; }
 
 docker run -it \
+    --rm \
```

Comment:
AM I mistaken or the `--rm` will force only one execution of this?

Reply:
@augustohp 

When we execute the command `docker ps`, it is listed all the containers that are running.

When we run `docker ps -a`, the command lists all containers that are running and all that are dead.

When we want to execute a container as a command, just to get an output, we can pass the --rm flag, because soon after execution it is excluded.

I do not know if my explanation was good.

Here's a link about it.
https://docs.docker.com/engine/reference/run/#clean-up---rm

**Note**: This repository helped me a lot and gave me lots of ideas :grin:

Reply:
The explanation was awesome! 🙇 

> **Note**: This repository helped me a lot and gave me lots of ideas 😁

Sweet! I use a similar one for [php](https://github.com/augustohp/ore-docker-php), and there is another [repository with my initial idea](https://github.com/augustohp/ore/issues/1) if it is of interest to you. 😉 

PS: Thanks for the improvements!